undefined reference to oboe::AudioStream::open

Build failing with:
oboe.dir/src/main/cpp/PlayAudioEngine.cpp.o CMakeFiles/hello-oboe.dir/src/main/cpp/SoundGenerator.cpp.o -landroid -llog oboe-bin/liboboe.a -llog -lOpenSLES -latomic -lm && :
/home/travis/build/google/oboe/src/opensles/AudioStreamOpenSLES.cpp:83: error: undefined reference to 'oboe::AudioStream::open()'
/home/travis/build/google/oboe/src/aaudio/AudioStreamAAudio.cpp:123: error: undefined reference to 'oboe::AudioStream::open

I see that Travis is saying all checks passed for pull requests.
https://travis-ci.org/google/oboe/builds/560726570

But in the details I can see the link error!

First Travis failure at:  #577

--- a/include/oboe/AudioStream.h
+++ b/include/oboe/AudioStream.h
@@ -63,7 +63,9 @@ public:
      *
      * @return
      */
-    virtual Result open() = 0;
+    virtual Result open() {
+        return Result::OK; // Called by subclasses. Might do more in the future.
+    }
 
     /**
      * Close the stream and deallocate any resources from the open() call.

